securedrop/static/js/journalist.js

```diff
@@ -209,4 +209,49 @@ ready(function() {
       });
   }
 
+  let deleteSourcesButton = document.getElementById('btn-delete-sources');
+  if (deleteSourcesButton) {
+    deleteSourcesButton.onclick = function() {
+      var checkboxes = document.querySelectorAll('input[name="cols_selected"]:checked');
```

Comment:
Is there a reason for `var` here? Seems like you're not accessing it outside of its declared scope.

Reply:
fixed.